packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/Result.js

```diff
@@ -22,24 +20,29 @@ function StepResult({ form }) {
   const initialSituations = getSituationsFor(data.situations, { idcc });
   const possibleSituations = filterSituations(initialSituations, criteria);
 
-  if (!possibleSituations.length && isNotYetProcessed(data.situations, idcc)) {
-    return (
-      <>
-        <Toast>
-          Nous n’avons pas encore traité cette convention collective. Le code du
-          travail ne prévoyant pas de durée précise du préavis de démission,
-          nous vous invitons à consulter le contenu de la convention collective.
-          <br />
-          <Link
-            href="/fiche-service-public/[slug]"
-            as={`/fiche-service-public/${ccn.slug}`}
-          >
-            <a>{ccn.title}</a>
-          </Link>
-        </Toast>
-      </>
-    );
-  }
+  /**
```

Comment:
Should we remove the code and comments ?

Reply:
removed